resources/dicts/patrols/general/border.json

```
@@ -2462,8 +2481,27 @@
                     }
 		],
 	    "weight": 10,
-            "art": "gen_bord_newcatkittypet1"
-	}
+            "art": "gen_kittypet_friends_with_dog1"
+	},
+    {
+        "text": "p_l lets out a vicious yowl, ignoring the now shrill voice calling out to {PRONOUN/p_l/object}, pleading with her to stop as {PRONOUN/p_l/subject} {VERB/p_l/attack/attacks} the dog, violently tearing at the dog's pelt as it yaps and whines. After the tussle, the dog hauls itself towards the now teary-eyed kittypet, and p_l can't help but smirk at {PRONOUN/n_c:0/poss} terrified expression.",
+                "exp": 5,
+            "new_cat": [
+            ["kittypet", "meeting", "exists"]
+            ],
+            "relationships": [
+                    {
+                            "cats_to": ["p_l"],
+                            "cats_from": ["n_c:0"],
+                            "mutual": true,
+                            "values": ["dislike"],
+                            "amount": 20
+                        }
+            ],
+            "weight": 10,
+            "stat_trait": ["bloodthirsty"],
```

```suggestion
            "stat_trait": ["bloodthirsty", "vengeful"],
```
I think this is probably pretty fitting as well